extensions/wikia/CreatePage/CreatePage.i18n.php

```diff
@@ -343,44 +342,36 @@
 ==Nadpis části==
 
 Sem napište druhou část stránky.',
-];
+);
 
-$messages['de'] = [
-	'createpage-desc' => 'Erlaubt das Erstellen von neuen Seiten mit Wikias WYSIWYG-Editor',
+$messages['de'] = array(
+	'createpage-desc' => 'Erlaubt das Erstellen von neuen Seiten mit Fandoms WYSIWYG-Editor',
 	'createpage-sp-title' => 'Neue Seite anlegen',
 	'createpage_title' => 'Neue Seite anlegen',
-	'createpage_title_caption' => 'Seitentitel:',
+	'createpage_title_caption' => 'Titel:',
 	'createpage_enter_text' => 'Text:',
-	'createpage_empty_title_error' => 'Titel erforderlich',
-	'createpage_empty_article_body_error' => 'Artikeltext erforderlich',
+	'createpage_empty_title_error' => 'Please write a title for your new article!',
```

Comment:
It was translated before, why do we make it English? There are multiple cases of this.